tiledb/sm/array_schema/dimension.cc

```diff
@@ -813,23 +818,29 @@ void Dimension::relevant_ranges(
   }
 }
 
-std::vector<bool> Dimension::covered_vec(
+tdb::pmr::vector<bool> Dimension::covered_vec(
     const NDRange& ranges,
     const Range& mbr,
-    const std::vector<uint64_t>& relevant_ranges) const {
+    const tdb::pmr::vector<uint64_t>& relevant_ranges) const {
   assert(covered_vec_func_ != nullptr);
-  return covered_vec_func_(ranges, mbr, relevant_ranges);
+  auto ret = covered_vec_func_(ranges, mbr, relevant_ranges);
+  return {
```

Comment:
Each time we do this, its a copy which seems not awesome.

Reply:
Removed